### PR TITLE
chore: replace auto-assign-reviewer-by-files action with CODEOWNERS

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

**Review by rinaldodev — APPROVED** [review]

**Review by rinaldodev — CHANGES_REQUESTED** [review]
Invalid owner on line 12:
cos-fleetshard-api/**/*" @rinaldodev @valdar @Naros @rk3rn3r

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# common
+cos-build-tools/**/* @rinaldodev @valdar @Naros @rk3rn3r
+cos-fleetshard-support/**/* @rinaldodev @valdar @Naros @rk3rn3r
+cos-fleetshard-operator/**/* @rinaldodev @valdar @Naros @rk3rn3r
+cos-fleetshard-operator-it/**/* @rinaldodev @valdar @Naros @rk3rn3r
+cos-fleetshard-operator-e2e/**/* @rinaldodev @valdar @Naros @rk3rn3r
+cos-fleetshard-it/**/* @rinaldodev @valdar @Naros @rk3rn3r
+
+# api
+cos-fleetshard-api/**/*" @rinaldodev @valdar @Naros @rk3rn3r
```
Comment (rk3rn3r) [inline .github/CODEOWNERS:12]: Invalid `"` char

Comment (rk3rn3r) [reply .github/CODEOWNERS:12]:
```suggestion
cos-fleetshard-api/**/* @rinaldodev @valdar @Naros @rk3rn3r
```

**Review by lgarciaaco — APPROVED** [review]

**Comment by lburgazzoli** [comment]
> GitHub is complaning:
> 
> ```
> Invalid owner on line 12:
> cos-fleetshard-api/**/*" @rinaldodev @valdar @Naros @rk3rn3r
> ```

done

**Review by rinaldodev — APPROVED** [review]

**Pull request merged**